### DISCUSSION TITLE
Is using sharp in Deno and Bun not possible?

### DISCUSSION THREAD

**Discussion by joshuaavalon** [windmill-labs/windmill#2843] [Questions] [answered]
I tried to use [sharp](https://github.com/lovell/sharp) in either Deno or Bun but both of them failed to install. Is it not possible to use it in windmill environment?

**Comment by rubenfiszel** [comment, marked as answer]
@joshuaavalon I remember a user asking about this on Discord and the conclusion was that sharp relied on lib that needs to be preinstalled on the worker. Which mean you need to either modify the base image of windmill OR use init scripts (https://www.windmill.dev/docs/advanced/preinstall_binaries#init-scripts). Highly recommend going the init scripts route.
